tests/tests_pytorch/strategies/test_common.py

```diff
@@ -54,7 +54,6 @@ def test_evaluate(tmpdir, trainer_kwargs):
     torch.testing.assert_close(old_weights, new_weights)
 
 
-@RunIf(min_torch="1.13")
```

Comment:
Did you search for `max_torch="1.12"` too?

Reply:
Yes there don't appear to be any

Reply:
docker workflows were merged into one: https://github.com/Lightning-AI/pytorch-lightning/blob/master/.github/workflows/docker-build.yml